packages/node_modules/samples/browser-meetings/app.js

```diff
@@ -261,6 +261,24 @@ document.getElementById('credentials').addEventListener('submit', (event) => {
   connect();
 });
 
+document.getElementById('PMR').addEventListener('submit', (event) => {
+  event.preventDefault();
+  return spark.meetings.personalMeetingRoom.getPersonalMeetingRoom().then((pmr) => {
+    document.getElementById('pmr-sip').value = spark.meetings.personalMeetingRoom.sipUri;
+    return spark.meetings.create(document.getElementById('pmr-sip').value).then((m) => {
+      meeting = m;
+      meeting.on('media:ready', (media) => {
```

Comment:
If we are going to have more of these (convo, etc) then we should move the connection/creation logic to a helper method so we don't have to write the same setup/tear down logic every time

Reply:
+1 will change when i do it for convo

Reply:
ahh nvm it was bothering me too changing now